### PR TITLE
Pawelka/jquery

### PR DESCRIPTION
**Pull request by moozzyk**

### CODE REVIEW

File: src/Microsoft.AspNet.SignalR.Client.JS/jquery.signalR.core.js [hunk 1]
```diff
@@ -249,7 +249,12 @@
         }
     };
 
-    _pageWindow.load(function () { _pageLoaded = true; });
+    if (typeof _pageWindow.on == "function") {
+        _pageWindow.on("load", function () { _pageLoaded = true; });
```
Comment (halter73) [inline src/Microsoft.AspNet.SignalR.Client.JS/jquery.signalR.core.js]: I take it that `.on` isn't provided by jQuery 1.6.4, but `.load` isn't provided by 3.0.0? Might deserve a comment.

**Comment by halter73** [comment]
:shipit:

File: src/Microsoft.AspNet.SignalR.Client.JS/jquery.signalR.core.js (hunk 1 above)
Comment (moozzyk) [inline src/Microsoft.AspNet.SignalR.Client.JS/jquery.signalR.core.js]: This is correct. I will add a comment.

**Pull request closed without merging**